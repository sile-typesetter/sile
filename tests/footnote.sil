\begin[papersize=a6,class=book]{document}
\nofolios
\script[src=packages/lorem]
\font[family=Gentium Plus,size=10pt]
\footnote:separator{\em{Separator}\skip[height=5mm]}
\footnote:options[interInsertionSkip=3mm]
Paragraph one.\footnote{Footnote one}

Paragraph two.\footnote{Footnote two \lorem[words=30]}

\lorem[words=60]

Paragraph three.\footnote{Footnote three \lorem[words=30]}

Paragraph four.\footnote{Footnote four \lorem[words=30]}

Paragraph five.\footnote{Footnote five \lorem[words=30]} \lorem[words=60]

Paragraph six.\footnote{Footnote six \lorem[words=30]}
\end{document}
