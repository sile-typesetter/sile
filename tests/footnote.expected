Set paper size 	297.6377985	419.5275636
Begin page
Mx 	44.7039
My 	56.2336
Set font 	Gentium Plus;10;400;;normal;;LTR
T	51 68 85 68 74 85 68 83 75 w=42.6123	(Paragraph)
Mx 	89.9748
T	82 81 72 w=15.1660	(one)
Mx 	105.1408
T	17 w=2.2900	(.)
Mx 	107.4309
My 	53.0548
Set font 	Gentium Plus;6.81152;400;;normal;;LTR
T	20 w=3.1962	(1)
Mx 	44.7039
My 	69.3349
Set font 	Gentium Plus;10;400;;normal;;LTR
T	51 68 85 68 74 85 68 83 75 w=42.6123	(Paragraph)
Mx 	89.9748
T	87 90 82 w=15.5322	(two)
Mx 	105.5070
T	17 w=2.2900	(.)
Mx 	107.7970
My 	66.1562
Set font 	Gentium Plus;6.81152;400;;normal;;LTR
T	21 w=3.1962	(2)
Mx 	44.7039
My 	82.4362
Set font 	Gentium Plus;10;400;;normal;;LTR
T	79 82 85 72 80 w=24.3506	(lorem)
Mx 	72.7093
T	76 83 86 88 80 w=25.2002	(ipsum)
Mx 	101.5643
T	71 82 79 82 85 w=21.9287	(dolor)
Mx 	127.1477
T	86 76 87 w=10.0146	(sit)
Mx 	140.8172
T	68 80 72 87 w=20.6836	(amet)
Mx 	165.1555
T	70 82 81 86 72 87 72 87 88 85 w=44.1602	(consetetur)
Mx 	212.9705
T	86 68 71 76 83 86 70 76 81 74 w=42.9980	(sadipscing)
Mx 	24.7039
My 	94.9869
T	72 79 76 87 85 w=17.4414	(elitr)
Mx 	45.9722
T	86 72 71 w=13.6816	(sed)
Mx 	63.4808
T	71 76 68 80 w=20.5322	(diam)
Mx 	87.8399
T	81 82 81 88 80 92 w=34.3262	(nonumy)
Mx 	125.9930
T	72 76 85 80 82 71 w=29.5508	(eirmod)
Mx 	159.3706
T	87 72 80 83 82 85 w=30.3809	(tempor)
Mx 	193.5784
T	76 81 89 76 71 88 81 87 w=35.3271	(invidunt)
Mx 	232.7324
T	88 87 w=8.7402	(ut)
Mx 	245.2996
T	79 68 w=7.2998	(la)
Mx 	252.5994
T	16 w=3.3691	(-)
Mx 	24.7039
My 	107.5376
T	69 82 85 72 w=18.6865	(bore)
Mx 	46.5063
T	72 87 w=8.0615	(et)
Mx 	57.6837
T	71 82 79 82 85 72 w=26.5479	(dolore)
Mx 	87.3473
T	80 68 74 81 68 w=27.6074	(magna)
Mx 	118.0706
T	68 79 76 84 88 92 68 80 w=38.0127	(aliquyam)
Mx 	159.1991
T	72 85 68 87 w=16.6113	(erat)
Mx 	178.9263
T	86 72 71 w=13.6816	(sed)
Mx 	195.7238
T	71 76 68 80 w=20.5322	(diam)
Mx 	219.3718
T	89 82 79 88 83 87 88 68 w=36.5967	(voluptua)
Mx 	24.7039
My 	120.0883
T	68 87 w=8.0322	(at)
Mx 	35.4208
T	89 72 85 82 w=18.5400	(vero)
Mx 	56.6455
T	72 82 86 w=13.5107	(eos)
Mx 	72.8409
T	72 87 w=8.0615	(et)
Mx 	83.5871
T	68 70 70 88 86 68 80 w=35.1123	(accusam)
Mx 	121.3840
T	72 87 w=8.0615	(et)
Mx 	132.1302
T	77 88 86 87 82 w=20.2197	(justo)
Mx 	155.0346
T	71 88 82 w=15.5273	(duo)
Mx 	173.2466
T	71 82 79 82 85 72 86 w=30.4102	(dolores)
Mx 	206.3414
T	72 87 w=8.0615	(et)
Mx 	217.0876
T	72 68 w=9.2090	(ea)
Mx 	228.9812
T	85 72 69 88 80 w=26.9873	(rebum)
Mx 	24.7039
My 	132.6389
T	86 87 72 87 w=15.3662	(stet)
Mx 	42.6114
T	70 79 76 87 68 w=17.8223	(clita)
Mx 	62.9749
T	78 68 86 71 w=18.5303	(kasd)
Mx 	84.0464
T	74 88 69 72 85 74 85 72 81 w=42.8076	(gubergren)
Mx 	129.3952
T	81 82 w=10.5469	(no)
Mx 	142.4833
T	86 72 68 w=13.0713	(sea)
Mx 	158.0958
T	87 68 78 76 80 68 87 68 w=36.2744	(takimata)
Mx 	196.9114
T	86 68 81 70 87 88 86 w=30.9424	(sanctus)
Mx 	230.3951
T	72 86 87 w=11.9238	(est)
Mx 	244.8601
T	79 82 w=7.7393	(lo)
Mx 	252.5994
T	16 w=3.3691	(-)
Mx 	24.7039
My 	145.1896
T	85 72 80 w=16.6113	(rem)
Mx 	45.1651
T	76 83 86 88 80 w=25.2002	(ipsum)
Mx 	74.2151
T	71 82 79 82 85 w=21.9287	(dolor)
Mx 	99.9936
T	86 76 87 w=10.0146	(sit)
Mx 	113.8581
T	68 80 72 87 w=20.6836	(amet)
Mx 	138.3915
T	79 82 85 72 80 w=24.3506	(lorem)
Mx 	166.5919
T	76 83 86 88 80 w=25.2002	(ipsum)
Mx 	195.6419
T	71 82 79 82 85 w=21.9287	(dolor)
Mx 	221.4204
T	86 76 87 w=10.0146	(sit)
Mx 	235.2849
T	68 80 72 87 w=20.6836	(amet)
Mx 	24.7039
My 	157.7403
T	70 82 81 86 72 87 72 87 88 85 w=44.1602	(consetetur)
Mx 	71.5157
T	86 68 71 76 83 86 70 76 81 74 w=42.9980	(sadipscing)
Mx 	117.1653
T	72 79 76 87 85 w=17.4414	(elitr)
Mx 	137.2583
T	86 72 71 w=13.6816	(sed)
Mx 	153.5915
T	71 76 68 80 w=20.5322	(diam)
Mx 	44.7039
My 	170.8416
T	51 68 85 68 74 85 68 83 75 w=42.6123	(Paragraph)
Mx 	89.9741
T	87 75 85 72 72 w=22.1582	(three)
Mx 	112.1323
T	17 w=2.2900	(.)
Mx 	114.4223
My 	167.6629
Set font 	Gentium Plus;6.81152;400;;normal;;LTR
T	22 w=3.1962	(3)
Mx 	44.7039
My 	183.9429
Set font 	Gentium Plus;10;400;;normal;;LTR
T	51 68 85 68 74 85 68 83 75 w=42.6123	(Paragraph)
Mx 	89.9746
T	73 82 88 85 w=17.3975	(four)
Mx 	107.3720
T	17 w=2.2900	(.)
Mx 	109.6621
My 	180.7642
Set font 	Gentium Plus;6.81152;400;;normal;;LTR
T	23 w=3.1962	(4)
Mx 	44.7039
My 	192.6832
Set font 	Gentium Plus;10;400;Italic;normal;;LTR
T	54 72 83 68 85 68 87 82 85 w=36.4258	(Separator)
Mx 	24.7039
My 	214.9228
Set font 	Gentium Plus;9;400;;normal;;LTR
T	20 w=4.2231	(1)
Mx 	28.9271
T	17 w=2.0610	(.)
Mx 	48.9881
T	41 82 82 87 81 82 87 72 w=33.2007	(Footnote)
Mx 	84.5828
T	82 81 72 w=13.6494	(one)
Mx 	24.7039
My 	230.2383
T	21 w=4.2231	(2)
Mx 	28.9271
T	17 w=2.0610	(.)
Mx 	48.9881
T	41 82 82 87 81 82 87 72 w=33.2007	(Footnote)
Mx 	84.3710
T	87 90 82 w=13.9790	(two)
Mx 	100.5322
T	79 82 85 72 80 w=21.9155	(lorem)
Mx 	124.6299
T	76 83 86 88 80 w=22.6802	(ipsum)
Mx 	149.4922
T	71 82 79 82 85 w=19.7358	(dolor)
Mx 	171.4102
T	86 76 87 w=9.0132	(sit)
Mx 	182.6056
T	68 80 72 87 w=18.6152	(amet)
Mx 	203.4030
T	70 82 81 86 72 87 72 87 88 85 w=39.7441	(consetetur)
Mx 	245.3293
T	86 68 w=7.6069	(sa)
Mx 	252.9363
T	16 w=3.0322	(-)
Mx 	24.7039
My 	241.0383
T	71 76 83 w=11.8872	(dip)
Mx 	36.5911
T	86 70 76 81 74 w=19.2041	(scing)
Mx 	57.9718
T	72 79 76 87 85 w=15.6973	(elitr)
Mx 	75.8456
T	86 72 71 w=12.3135	(sed)
Mx 	90.3356
T	71 76 68 80 w=18.4790	(diam)
Mx 	110.9911
T	81 82 81 88 80 92 w=30.8936	(nonumy)
Mx 	144.0612
T	72 76 85 80 82 71 w=26.5957	(eirmod)
Mx 	172.8334
T	87 72 80 83 82 85 w=27.3428	(tempor)
Mx 	202.3527
T	76 81 89 76 71 88 81 87 w=31.7944	(invidunt)
Mx 	236.3237
T	88 87 w=7.8662	(ut)
Mx 	246.3665
T	79 68 w=6.5698	(la)
Mx 	252.9363
T	16 w=3.0322	(-)
Mx 	24.7039
My 	251.8383
T	69 82 85 72 w=16.8179	(bore)
Mx 	43.6864
T	72 87 w=7.2554	(et)
Mx 	53.1063
T	71 82 79 82 85 72 w=23.8931	(dolore)
Mx 	79.1639
T	80 68 74 81 68 w=24.8467	(magna)
Mx 	106.1751
T	68 79 76 84 88 92 68 80 w=34.2114	(aliquyam)
Mx 	142.5511
T	72 85 68 87 w=14.9502	(erat)
Mx 	159.6658
T	86 72 71 w=12.3135	(sed)
Mx 	174.1438
T	71 76 68 80 w=18.4790	(diam)
Mx 	194.7874
T	89 82 79 88 83 87 88 68 w=32.9370	(voluptua)
Mx 	229.8889
T	68 87 w=7.2290	(at)
Mx 	239.2825
T	89 72 85 82 w=16.6860	(vero)
Mx 	24.7039
My 	262.6383
T	72 82 86 w=12.1597	(eos)
Mx 	39.2584
T	72 87 w=7.2554	(et)
Mx 	48.9085
T	68 70 70 88 86 68 80 w=31.6011	(accusam)
Mx 	82.9043
T	72 87 w=7.2554	(et)
Mx 	24.7039
My 	277.9537
T	22 w=4.2231	(3)
Mx 	28.9271
T	17 w=2.0610	(.)
Mx 	48.9881
T	41 82 82 87 81 82 87 72 w=33.2007	(Footnote)
Mx 	85.3507
T	87 75 85 72 72 w=19.9424	(three)
Mx 	108.4550
T	79 82 85 72 80 w=21.9155	(lorem)
Mx 	133.5324
T	76 83 86 88 80 w=22.6802	(ipsum)
Mx 	159.3744
T	71 82 79 82 85 w=19.7358	(dolor)
Mx 	182.2722
T	86 76 87 w=9.0132	(sit)
Mx 	194.4472
T	68 80 72 87 w=18.6152	(amet)
Mx 	216.2244
T	70 82 81 86 72 87 72 87 88 85 w=39.7441	(consetetur)
Mx 	24.7039
My 	288.7537
T	86 68 71 76 83 86 70 76 81 74 w=38.6982	(sadipscing)
Mx 	66.1002
T	72 79 76 87 85 w=15.6973	(elitr)
Mx 	84.4954
T	86 72 71 w=12.3135	(sed)
Mx 	99.5069
T	71 76 68 80 w=18.4790	(diam)
Mx 	120.6839
T	81 82 81 88 80 92 w=30.8936	(nonumy)
Mx 	154.2754
T	72 76 85 80 82 71 w=26.5957	(eirmod)
Mx 	183.5691
T	87 72 80 83 82 85 w=27.3428	(tempor)
Mx 	213.6099
T	76 81 89 76 71 88 81 87 w=31.7944	(invidunt)
Mx 	248.1023
T	88 87 w=7.8662	(ut)
Mx 	24.7039
My 	299.5537
T	79 68 69 82 85 72 w=23.3877	(labore)
Mx 	51.6207
T	72 87 w=7.2554	(et)
Mx 	62.4051
T	71 82 79 82 85 72 w=23.8931	(dolore)
Mx 	89.8273
T	80 68 74 81 68 w=24.8467	(magna)
Mx 	118.2030
T	68 79 76 84 88 92 68 80 w=34.2114	(aliquyam)
Mx 	155.9435
T	72 85 68 87 w=14.9502	(erat)
Mx 	174.4228
T	86 72 71 w=12.3135	(sed)
Mx 	190.2653
T	71 76 68 80 w=18.4790	(diam)
Mx 	212.2734
T	89 82 79 88 83 87 88 68 w=32.9370	(voluptua)
Mx 	248.7395
T	68 87 w=7.2290	(at)
Mx 	24.7039
My 	310.3537
T	89 72 85 82 w=16.6860	(vero)
Mx 	43.7828
T	72 82 86 w=12.1597	(eos)
Mx 	58.3354
T	72 87 w=7.2554	(et)
Mx 	67.9836
T	68 70 70 88 86 68 80 w=31.6011	(accusam)
Mx 	101.9775
T	72 87 w=7.2554	(et)
Mx 	24.7039
My 	325.6692
T	23 w=4.2231	(4)
Mx 	28.9271
T	17 w=2.0610	(.)
Mx 	48.9881
T	41 82 82 87 81 82 87 72 w=33.2007	(Footnote)
Mx 	84.1611
T	73 82 88 85 w=15.6577	(four)
Mx 	101.7912
T	79 82 85 72 80 w=21.9155	(lorem)
Mx 	125.6791
T	76 83 86 88 80 w=22.6802	(ipsum)
Mx 	150.3316
T	71 82 79 82 85 w=19.7358	(dolor)
Mx 	172.0398
T	86 76 87 w=9.0132	(sit)
Mx 	183.0253
T	68 80 72 87 w=18.6152	(amet)
Mx 	203.6129
T	70 82 81 86 72 87 72 87 88 85 w=39.7441	(consetetur)
Mx 	245.3293
T	86 68 w=7.6069	(sa)
Mx 	252.9363
T	16 w=3.0322	(-)
Mx 	24.7039
My 	336.4692
T	71 76 83 w=11.8872	(dip)
Mx 	36.5911
T	86 70 76 81 74 w=19.2041	(scing)
Mx 	57.9718
T	72 79 76 87 85 w=15.6973	(elitr)
Mx 	75.8456
T	86 72 71 w=12.3135	(sed)
Mx 	90.3356
T	71 76 68 80 w=18.4790	(diam)
Mx 	110.9911
T	81 82 81 88 80 92 w=30.8936	(nonumy)
Mx 	144.0612
T	72 76 85 80 82 71 w=26.5957	(eirmod)
Mx 	172.8334
T	87 72 80 83 82 85 w=27.3428	(tempor)
Mx 	202.3527
T	76 81 89 76 71 88 81 87 w=31.7944	(invidunt)
Mx 	236.3237
T	88 87 w=7.8662	(ut)
Mx 	246.3665
T	79 68 w=6.5698	(la)
Mx 	252.9363
T	16 w=3.0322	(-)
Mx 	24.7039
My 	347.2692
T	69 82 85 72 w=16.8179	(bore)
Mx 	43.6864
T	72 87 w=7.2554	(et)
Mx 	53.1063
T	71 82 79 82 85 72 w=23.8931	(dolore)
Mx 	79.1639
T	80 68 74 81 68 w=24.8467	(magna)
Mx 	106.1751
T	68 79 76 84 88 92 68 80 w=34.2114	(aliquyam)
Mx 	142.5511
T	72 85 68 87 w=14.9502	(erat)
Mx 	159.6658
T	86 72 71 w=12.3135	(sed)
Mx 	174.1438
T	71 76 68 80 w=18.4790	(diam)
Mx 	194.7874
T	89 82 79 88 83 87 88 68 w=32.9370	(voluptua)
Mx 	229.8889
T	68 87 w=7.2290	(at)
Mx 	239.2825
T	89 72 85 82 w=16.6860	(vero)
New page
Mx 	61.6693
My 	56.2336
Set font 	Gentium Plus;10;400;;normal;;LTR
T	51 68 85 68 74 85 68 83 75 w=42.6123	(Paragraph)
Mx 	106.9744
T	737 89 72 w=15.3711	(five)
Mx 	122.3455
T	17 w=2.2900	(.)
Mx 	124.6355
My 	53.0548
Set font 	Gentium Plus;6.81152;400;;normal;;LTR
T	24 w=3.1962	(5)
Mx 	130.5246
My 	56.2336
Set font 	Gentium Plus;10;400;;normal;;LTR
T	79 82 85 72 80 w=24.3506	(lorem)
Mx 	157.5679
T	76 83 86 88 80 w=25.2002	(ipsum)
Mx 	185.4609
T	71 82 79 82 85 w=21.9287	(dolor)
Mx 	210.0824
T	86 76 87 w=10.0146	(sit)
Mx 	222.7899
T	68 80 72 87 w=20.6836	(amet)
Mx 	246.1663
T	70 82 81 w=14.9170	(con)
Mx 	261.0833
T	86 72 w=8.4814	(se)
Mx 	269.5647
T	16 w=3.3691	(-)
Mx 	41.6693
My 	68.2336
T	87 72 w=8.0615	(te)
Mx 	49.7308
T	87 88 85 w=12.7002	(tur)
Mx 	65.5155
T	86 68 71 76 83 86 70 76 81 74 w=42.9980	(sadipscing)
Mx 	111.5981
T	72 79 76 87 85 w=17.4414	(elitr)
Mx 	132.1241
T	86 72 71 w=13.6816	(sed)
Mx 	148.8902
T	71 76 68 80 w=20.5322	(diam)
Mx 	172.5070
T	81 82 81 88 80 92 w=34.3262	(nonumy)
Mx 	209.9177
T	72 76 85 80 82 71 w=29.5508	(eirmod)
Mx 	242.5530
T	87 72 80 83 82 85 w=30.3809	(tempor)
Mx 	41.6693
My 	80.2336
T	76 81 89 76 71 88 81 87 w=35.3271	(invidunt)
Mx 	80.8325
T	88 87 w=8.7402	(ut)
Mx 	93.4088
T	79 68 69 82 85 72 w=25.9863	(labore)
Mx 	123.2312
T	72 87 w=8.0615	(et)
Mx 	135.1287
T	71 82 79 82 85 72 w=26.5479	(dolore)
Mx 	165.5126
T	80 68 74 81 68 w=27.6074	(magna)
Mx 	196.9561
T	68 79 76 84 88 92 68 80 w=38.0127	(aliquyam)
Mx 	238.8048
T	72 85 68 87 w=16.6113	(erat)
Mx 	259.2522
T	86 72 71 w=13.6816	(sed)
Mx 	41.6693
My 	92.2336
T	71 76 68 80 w=20.5322	(diam)
Mx 	64.7748
T	89 82 79 88 83 87 88 68 w=36.5967	(voluptua)
Mx 	103.9447
T	68 87 w=8.0322	(at)
Mx 	114.5501
T	89 72 85 82 w=18.5400	(vero)
Mx 	135.6634
T	72 82 86 w=13.5107	(eos)
Mx 	151.7474
T	72 87 w=8.0615	(et)
Mx 	162.3821
T	68 70 70 88 86 68 80 w=35.1123	(accusam)
Mx 	200.0677
T	72 87 w=8.0615	(et)
Mx 	210.7024
T	77 88 86 87 82 w=20.2197	(justo)
Mx 	233.4954
T	71 88 82 w=15.5273	(duo)
Mx 	251.5960
T	71 82 w=10.2295	(do)
Mx 	261.8255
T	79 82 w=7.7393	(lo)
Mx 	269.5647
T	16 w=3.3691	(-)
Mx 	41.6693
My 	104.2336
T	85 72 86 w=12.4414	(res)
Mx 	56.1253
T	72 87 w=8.0615	(et)
Mx 	66.2015
T	72 68 w=9.2090	(ea)
Mx 	77.4251
T	85 72 69 88 80 w=26.9873	(rebum)
Mx 	106.4271
T	86 87 72 87 w=15.3662	(stet)
Mx 	123.8079
T	70 79 76 87 68 w=17.8223	(clita)
Mx 	143.6448
T	78 68 86 71 w=18.5303	(kasd)
Mx 	164.1897
T	74 88 69 72 85 74 85 72 81 w=42.8076	(gubergren)
Mx 	209.0120
T	81 82 w=10.5469	(no)
Mx 	221.5735
T	86 72 68 w=13.0713	(sea)
Mx 	236.6594
T	87 68 78 76 80 68 87 68 w=36.2744	(takimata)
Mx 	41.6693
My 	116.2336
T	86 68 81 70 87 88 86 w=30.9424	(sanctus)
Mx 	75.1751
T	72 86 87 w=11.9238	(est)
Mx 	89.6624
T	79 82 85 72 80 w=24.3506	(lorem)
Mx 	116.5765
T	76 83 86 88 80 w=25.2002	(ipsum)
Mx 	144.3402
T	71 82 79 82 85 w=21.9287	(dolor)
Mx 	168.8323
T	86 76 87 w=10.0146	(sit)
Mx 	181.4105
T	68 80 72 87 w=20.6836	(amet)
Mx 	204.6575
T	79 82 85 72 80 w=24.3506	(lorem)
Mx 	231.5716
T	76 83 86 88 80 w=25.2002	(ipsum)
Mx 	259.3352
T	71 82 w=10.2295	(do)
Mx 	269.5647
T	16 w=3.3691	(-)
Mx 	41.6693
My 	128.2336
T	79 82 85 w=11.6992	(lor)
Mx 	56.0146
T	86 76 87 w=10.0146	(sit)
Mx 	68.6753
T	68 80 72 87 w=20.6836	(amet)
Mx 	92.0049
T	70 82 81 86 72 87 72 87 88 85 w=44.1602	(consetetur)
Mx 	138.8111
T	86 68 71 76 83 86 70 76 81 74 w=42.9980	(sadipscing)
Mx 	184.4552
T	72 79 76 87 85 w=17.4414	(elitr)
Mx 	204.5427
T	86 72 71 w=13.6816	(sed)
Mx 	220.8704
T	71 76 68 80 w=20.5322	(diam)
Mx 	61.6693
My 	140.2336
T	51 68 85 68 74 85 68 83 75 w=42.6123	(Paragraph)
Mx 	106.9406
T	86 76 91 w=11.6016	(six)
Mx 	118.5421
T	17 w=2.2900	(.)
Mx 	120.8322
My 	137.0548
Set font 	Gentium Plus;6.81152;400;;normal;;LTR
T	25 w=3.1962	(6)
Mx 	61.6693
My 	228.6863
Set font 	Gentium Plus;10;400;Italic;normal;;LTR
T	54 72 83 68 85 68 87 82 85 w=36.4258	(Separator)
Mx 	41.6693
My 	253.9037
Set font 	Gentium Plus;9;400;;normal;;LTR
T	72 82 86 w=12.1597	(eos)
Mx 	56.2237
T	72 87 w=7.2554	(et)
Mx 	65.8738
T	68 70 70 88 86 68 80 w=31.6011	(accusam)
Mx 	99.8697
T	72 87 w=7.2554	(et)
Mx 	41.6693
My 	269.2192
T	24 w=4.2231	(5)
Mx 	45.8924
T	17 w=2.0610	(.)
Mx 	65.9535
T	41 82 82 87 81 82 87 72 w=33.2007	(Footnote)
Mx 	101.3545
T	737 89 72 w=13.8340	(five)
Mx 	117.3888
T	79 82 85 72 80 w=21.9155	(lorem)
Mx 	141.5046
T	76 83 86 88 80 w=22.6802	(ipsum)
Mx 	166.3851
T	71 82 79 82 85 w=19.7358	(dolor)
Mx 	188.3212
T	86 76 87 w=9.0132	(sit)
Mx 	199.5347
T	68 80 72 87 w=18.6152	(amet)
Mx 	220.3503
T	70 82 81 86 72 87 72 87 88 85 w=39.7441	(consetetur)
Mx 	262.2947
T	86 68 w=7.6069	(sa)
Mx 	269.9016
T	16 w=3.0322	(-)
Mx 	41.6693
My 	280.0192
T	71 76 83 w=11.8872	(dip)
Mx 	53.5565
T	86 70 76 81 74 w=19.2041	(scing)
Mx 	74.9371
T	72 79 76 87 85 w=15.6973	(elitr)
Mx 	92.8109
T	86 72 71 w=12.3135	(sed)
Mx 	107.3009
T	71 76 68 80 w=18.4790	(diam)
Mx 	127.9565
T	81 82 81 88 80 92 w=30.8936	(nonumy)
Mx 	161.0266
T	72 76 85 80 82 71 w=26.5957	(eirmod)
Mx 	189.7988
T	87 72 80 83 82 85 w=27.3428	(tempor)
Mx 	219.3181
T	76 81 89 76 71 88 81 87 w=31.7944	(invidunt)
Mx 	253.2891
T	88 87 w=7.8662	(ut)
Mx 	263.3318
T	79 68 w=6.5698	(la)
Mx 	269.9016
T	16 w=3.0322	(-)
Mx 	41.6693
My 	290.8192
T	69 82 85 72 w=16.8179	(bore)
Mx 	60.6517
T	72 87 w=7.2554	(et)
Mx 	70.0716
T	71 82 79 82 85 72 w=23.8931	(dolore)
Mx 	96.1292
T	80 68 74 81 68 w=24.8467	(magna)
Mx 	123.1405
T	68 79 76 84 88 92 68 80 w=34.2114	(aliquyam)
Mx 	159.5164
T	72 85 68 87 w=14.9502	(erat)
Mx 	176.6312
T	86 72 71 w=12.3135	(sed)
Mx 	191.1092
T	71 76 68 80 w=18.4790	(diam)
Mx 	211.7527
T	89 82 79 88 83 87 88 68 w=32.9370	(voluptua)
Mx 	246.8543
T	68 87 w=7.2290	(at)
Mx 	256.2478
T	89 72 85 82 w=16.6860	(vero)
Mx 	41.6693
My 	301.6192
T	72 82 86 w=12.1597	(eos)
Mx 	56.2237
T	72 87 w=7.2554	(et)
Mx 	65.8738
T	68 70 70 88 86 68 80 w=31.6011	(accusam)
Mx 	99.8697
T	72 87 w=7.2554	(et)
Mx 	41.6693
My 	316.9346
T	25 w=4.2231	(6)
Mx 	45.8924
T	17 w=2.0610	(.)
Mx 	65.9535
T	41 82 82 87 81 82 87 72 w=33.2007	(Footnote)
Mx 	101.7785
T	86 76 91 w=10.4414	(six)
Mx 	114.8443
T	79 82 85 72 80 w=21.9155	(lorem)
Mx 	139.3842
T	76 83 86 88 80 w=22.6802	(ipsum)
Mx 	164.6888
T	71 82 79 82 85 w=19.7358	(dolor)
Mx 	187.0490
T	86 76 87 w=9.0132	(sit)
Mx 	198.6866
T	68 80 72 87 w=18.6152	(amet)
Mx 	219.9262
T	70 82 81 86 72 87 72 87 88 85 w=39.7441	(consetetur)
Mx 	262.2947
T	86 68 w=7.6069	(sa)
Mx 	269.9016
T	16 w=3.0322	(-)
Mx 	41.6693
My 	327.7346
T	71 76 83 w=11.8872	(dip)
Mx 	53.5565
T	86 70 76 81 74 w=19.2041	(scing)
Mx 	74.9371
T	72 79 76 87 85 w=15.6973	(elitr)
Mx 	92.8109
T	86 72 71 w=12.3135	(sed)
Mx 	107.3009
T	71 76 68 80 w=18.4790	(diam)
Mx 	127.9565
T	81 82 81 88 80 92 w=30.8936	(nonumy)
Mx 	161.0266
T	72 76 85 80 82 71 w=26.5957	(eirmod)
Mx 	189.7988
T	87 72 80 83 82 85 w=27.3428	(tempor)
Mx 	219.3181
T	76 81 89 76 71 88 81 87 w=31.7944	(invidunt)
Mx 	253.2891
T	88 87 w=7.8662	(ut)
Mx 	263.3318
T	79 68 w=6.5698	(la)
Mx 	269.9016
T	16 w=3.0322	(-)
Mx 	41.6693
My 	338.5346
T	69 82 85 72 w=16.8179	(bore)
Mx 	60.6517
T	72 87 w=7.2554	(et)
Mx 	70.0716
T	71 82 79 82 85 72 w=23.8931	(dolore)
Mx 	96.1292
T	80 68 74 81 68 w=24.8467	(magna)
Mx 	123.1405
T	68 79 76 84 88 92 68 80 w=34.2114	(aliquyam)
Mx 	159.5164
T	72 85 68 87 w=14.9502	(erat)
Mx 	176.6312
T	86 72 71 w=12.3135	(sed)
Mx 	191.1092
T	71 76 68 80 w=18.4790	(diam)
Mx 	211.7527
T	89 82 79 88 83 87 88 68 w=32.9370	(voluptua)
Mx 	246.8543
T	68 87 w=7.2290	(at)
Mx 	256.2478
T	89 72 85 82 w=16.6860	(vero)
Mx 	41.6693
My 	349.3346
T	72 82 86 w=12.1597	(eos)
Mx 	56.2237
T	72 87 w=7.2554	(et)
Mx 	65.8738
T	68 70 70 88 86 68 80 w=31.6011	(accusam)
Mx 	99.8697
T	72 87 w=7.2554	(et)
End page
Finish
